styles/company-life.css

```diff
@@ -0,0 +1,177 @@
+.life-container {
+    background-color: #ffffff;
+    justify-content: flex-start;
+}
+
+.life {
+    margin: 54px 0 54px 0;
+    justify-content: flex-start;
+    width: 95%;
+}
+
+.life__header, life__body{
+    width: 100%;
+    text-align: center;
+}
+
+.life__header + .life__body {
+    margin-top: 40px;
+}
+
+.life__body {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;
+}
+
+.life__title {
+    text-align: center;
+    color: #003d48;
+}
+
+.life__prh {
+    font-size: 22px;
+    font-weight: 300;
+    color: #6dc5dd;
+    line-height: 46px;
+    height: 48px;
+
+    border-width: 1px 0 1px 0;
+    border-color: #ececec;
+    border-style: solid; 
+}
+
+.life__title + .life__prh {
+    margin-top: 25px;
+}
+
+.life__list + .life__list{
+    margin-left: 5%;
+}
+
+.life__list {
+    align-items: flex-start;
+}
+
+.life__says-list-item + .life__says-list-item {
+    margin-top: 40px;
+}
+
+.life__news-item + .life__news-item {
+    margin-top: 50px;
+}
+
+.news {
+    min-height: 90px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+}
+
+.news__date + .news__article{
+    margin-left: 28px;
+}
+
+.news__date {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex-shrink: 0;
+
+    background-color: #6dc5dd;
+    border-radius: 3px;
+    height: 68px;
+    width: 70px;
+    color: #ffffff;
+}
+
+.news__day, .news__month {
+    font-family: 'Source Sans Pro', sans-serif;;
+    width: 100%;
+    font-weight: bold;
+    text-align: center;
+}
+
+.news__day {
+    font-size: 35px;
+    line-height: 40px;
+}
+  
+.news__month {
+    font-size: 18px;
+    line-height: 21px;
+    border-top-width: 1px;
+    border-top-style: solid;
+}
+
+.news__article {
+    display: flex;
```

Comment:
For example, flexbox here is completely redundant. It doesn't place elements in some tricky order and placing elements in column is just a natural flow of elements in the documents.